docs/docs/guides/migrate/sources/keycloak.md

```diff
@@ -0,0 +1,455 @@
+
+In order to set up the ZITADEL sample Angular application, ensure you have [Node.js](https://nodejs.org/en/) installed and clone the example repository:
+
+```Bash Script
+git clone https://github.com/zitadel/zitadel-angular.git
+npm install -g @angular/cli
+npm install
+```
+
+In this tutorial, the Keycloak realm is named `my-realm`, and the client ID is `test-client`. Edit the [src/app/app.module.ts file](https://github.com/zitadel/zitadel-angular/blob/main/src/app/app.module.ts) and update the `client ID` and `issuer`:
+
+```JavaScript 
+const authConfig: AuthConfig = {
+   scope: 'openid profile email',
+   responseType: 'code',
+   oidc: true,
+   clientId: 'test-client',
+   issuer: 'http://localhost:8081/realms/my-realm',
+   redirectUri: 'http://localhost:4200/auth/callback',
+   postLogoutRedirectUri: 'http://localhost:4200/signedout',
+   requireHttps: false, // required for running locally
+   disableAtHashCheck: true
+};
+```
+
+The landing page of the application will be as follows: 
+
+
+Per the sample [documentation](https://github.com/zitadel/zitadel-angular#readme), running the development server will serve the browser-side application at [http://localhost:4200/](http://localhost:4200/):
+
+```Bash Script
+npm start
+```
+
+<img src="/docs/img/guides/migrate/keycloak-17.png" alt="Migrating users from Keycloak to ZITADEL"/>
+
+After clicking the **Authenticate** button, users will be redirected to the login page hosted in Keycloak:
+
+<img src="/docs/img/guides/migrate/keycloak-18.png" alt="Migrating users from Keycloak to ZITADEL"/>
+
+After successfully logging into Keycloak, users are redirected back to the sample application:
+
+
+Now, you've successfully created a sample application that uses Keycloak for user and session management.
+
+<img src="/docs/img/guides/migrate/keycloak-19.png" alt="Migrating users from Keycloak to ZITADEL"/>
+
+<img src="/docs/img/guides/migrate/keycloak-20.png" alt="Migrating users from Keycloak to ZITADEL"/>
+
+<img src="/docs/img/guides/migrate/keycloak-21.png" alt="Migrating users from Keycloak to ZITADEL"/>
+
+Now, you've successfully created a sample application that uses Keycloak for user and session management.
+
+## Set up ZITADEL
+
+After creating a sample application that connects to Keycloak, you need to set up ZITADEL in order to migrate the application and users from Keycloak to ZITADEL. For this, ZITADEL offers a [Docker Compose](https://zitadel.com/docs/self-hosting/deploy/compose) installation guide:
+
+```Bash Script
+# Download the docker compose example configuration.
+
+wget https://raw.githubusercontent.com/zitadel/zitadel/main/docs/docs/self-hosting/deploy/docker-compose.yaml
+
+
+# Run the database and application containers. It will start both the database and the web-server components
+docker compose up --detach
+```
```

Comment:
I would not duplicate the script here. Either pull it via embedded element or only reference to the documentation. We should not have to manage it in two places, in case something changes.

Reply:
Done.